ci/cluster-api/vsphere/templates/cluster.yaml

```diff
@@ -98,65 +44,106 @@ spec:
     spec:
       cloneMode: linkedClone
       datacenter: DATACENTERNAME
-      datastore: WorkloadDatastore
+      datastore: vsanDatastore
```

Comment:
Need to change to 208.1-20TB-NFS.

Later let's refactor it also to a variable.